### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by Kralizek** [AutoFixture/AutoFixture#1217] [Ideas]
Here we go me bouncing at the opportunities offered by the Discussions board.

How about setting up a Discord server where contributors can meet?

The only con I can see is that it would create some kind of ambiguity between what gets discussed in Discord and what should be a thread or an issue here.

**Comment by aivascu** [comment]
@Kralizek I was actually working on getting it setup right before GitHub announced Discussions. 
Honestly I was relieved by these news because, for some reason I couldn't get the audio calls and the moderation to work in Discord, and I wanted really bad to get some communication/feedback going. 

To me GitHub Discussions seems like good fit for this project. 
If you want to have a look at the Discord server I was working on, I could share the link.

As a software developer I am ashamed to fail so miserably at technology.
In my defense I don't use Discord at all, and I'm getting too old for this new kids stuff. 😅

**Reply by Kralizek** [reply]
Feel free to DM me the link over Twitter. I never really set up a server myself but I can gladly put some time in it.
